docs/example-scenario/data/big-data-with-iot.md

```diff
@@ -0,0 +1,90 @@
+* [Load Balancer](/azure/load-balancer) lets you scale your applications and create high availability for your services. The load balancer distributes data submissions from construction equipment devices.
+* [Azure Virtual Machines](/azure/virtual-machines) host the web services that receive and ingest the construction results data into the Apache Cassandra database.
+* [Apache Cassandra](http://cassandra.apache.org/) is a distributed NoSQL database used to store construction data for later processing by Apache Spark.
+* [Web Apps](/azure/app-service) hosts the end-user web application, which can be used to query and view source data and images. Users can also initiate batch jobs in Apache Spark via the application.
+* [Apache Spark on HDInsight](/azure/hdinsight/spark) supports in-memory processing to boost the performance of big-data analytic applications. In this scenario, Spark is used to run complex algorithms over the data stored in Apache Cassandra.
+
+### Alternatives
+
+* [Cosmos DB](/azure/cosmos-db) is an alternative NoSQL database technology. Cosmos DB provides [multi-master support at global scale](/azure/cosmos-db/multi-region-writers) with [multiple well-defined consistency levels](/azure/cosmos-db/consistency-levels) to meet various customer requirements. It also supports the [Cassandra API](/azure/cosmos-db/cassandra-introduction). 
+* [Data Lake Storage](/azure/storage/data-lake-storage) is an alternative to Blob storage. For this scenario, Data Lake Storage was not available in the targeted region.
+* [Web Apps](/azure/app-service) could also be used to host the web services for ingesting construction results data.
+* Many technology options are available for real-time message ingestion, data storage, stream processing, storage of analytical data, and analytics and reporting. For an overview of these options, their capabilities, and key selection criteria, see [Big data architectures: Real-time processing](/azure/architecture/data-guide/technology-choices/real-time-ingestion) in the [Azure Data Architecture Guide](/azure/architecture/data-guide/).
+   
+## Considerations
+
+The broad availability of Azure regions is an important factor for this scenario. Having more than one region in a single country can provide disaster recovery while also enabling compliance with contractual obligations and law enforcement requirements. Azure's high speed communication between regions is also an important factor in this scenario.
+
+Azure support for open source technologies allowed the customer to take advantage of their existing workforce skills. The customer can also accelerate the adoption of new technologies with significantly lower costs and operating workloads compared to an on-premises solution. 
+
+## Pricing
+
+The following considerations will drive a substantial portion of the costs for this solution.
```

Comment:
Can we stub out a calculator link with all of the resources used, even if it's just a $0 or $1 charge?

Reply:
I'll do that.